### ISSUE TITLE
Login api returns 500

### ISSUE THREAD

**Issue by abhn** [darth-dodo/gringotts-backend#31]
Login api returns 500 on sending a `post` request with Axios, Postman or Curl. I tried `application/json` and `application/x-www-form-encoded`.

#### Steps to reproduce
1. Send a post request to login endpoint with the credentials from the api docs or readme

#### Expected
API to return token

#### What happened instead
API returned 500 with rails error page

#### Debug and traceback
```
2019-07-30T06:43:13.682189+00:00 app[web.1]: F, [2019-07-30T06:43:13.682079 #4] FATAL -- : [1e1297c5-2d73-4c02-a8f0-1c8075d6976a]
2019-07-30T06:43:13.682252+00:00 app[web.1]: F, [2019-07-30T06:43:13.682199 #4] FATAL -- : [1e1297c5-2d73-4c02-a8f0-1c8075d6976a] NoMethodError (undefined method `id' for nil:NilClass):
2019-07-30T06:43:13.682313+00:00 app[web.1]: F, [2019-07-30T06:43:13.682250 #4] FATAL -- : [1e1297c5-2d73-4c02-a8f0-1c8075d6976a]
2019-07-30T06:43:13.682369+00:00 app[web.1]: F, [2019-07-30T06:43:13.682322 #4] FATAL -- : [1e1297c5-2d73-4c02-a8f0-1c8075d6976a] app/controllers/api_controller.rb:22:in `set_raven_context!'
```

https://github.com/darth-dodo/gringotts-backend/blob/master/app/controllers/api_controller.rb#L22

**Comment by darth-dodo** [comment]
Issued the fix in PR #33 

Great RCA @abhn

**Comment by abhn** [comment]
Thanks! Fixed
